### PR TITLE
fix(db/translation): Chinese translation of road signs

### PR DESCRIPTION
**Pull request by mpfans**
## Changes Proposed:
* Boat to Azuremyst
* Boat to Teldrassil
* Boat to Stormwind

## Tests Performed:

- tested ingame

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1654674618622504335.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Add Chinese translation of road signs
+DELETE FROM `gameobject_template_locale` WHERE `entry` IN (184084,176365,176364) AND `locale` = 'zhCN';
+INSERT INTO `gameobject_template_locale` (`entry`, `locale`, `name`, `castBarCaption`, `VerifiedBuild`) VALUES 
+(184084, 'zhCN', '开往秘蓝岛的船只', '', 18019),
+(176365, 'zhCN', '开往泰达希尔的船只', '', 18019);
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1654674618622504335.sql:5]: (176365, 'zhCN', '开往泰达希尔的船只', '', 18019); 
needs to be
(176365, 'zhCN', '开往泰达希尔的船只', '', 18019),

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1654674618622504335.sql (hunk 1 above)
Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1654674618622504335.sql]: i went ahead and changed it for you

**Review by acidmanifesto — APPROVED** [review]

**Comment by acidmanifesto** [comment]
Thank you for the PR

File: data/sql/updates/pending_db_world/rev_1654674618622504335.sql (hunk 1 above)
Comment (mpfans) [reply data/sql/updates/pending_db_world/rev_1654674618622504335.sql]: > i went ahead and changed it for you

ty

**Pull request merged**